common.gradle

```diff
@@ -81,35 +83,61 @@ if(!rootProject.hasProperty("useLocalDependencies")) {
 ext.gearvrfVersion='3.1.1-SNAPSHOT'
 project.ext.jomlVersion = "1.9.1-SNAPSHOT"
 
-dependencies {
-    compile "org.joml:joml-android:${jomlVersion}"
-    if(rootProject.useLocalDependencies) {
+if(rootProject.useLocalDependencies) {
+    dependencies {
+        compile "org.joml:joml-android:${jomlVersion}"
         compile(name: 'framework-debug', ext: 'aar')
-    } else {
-        compile "org.gearvrf:framework:$gearvrfVersion"
-    }
 
-    if (rootProject.hasProperty("only_daydream")) {
-        if(rootProject.useLocalDependencies) {
+        if (rootProject.hasProperty("only_daydream")) {
             compile(name: 'backend_daydream-debug', ext: 'aar')
+            compile 'com.google.vr:sdk-base:1.10.0'
+            compile 'com.google.protobuf.nano:protobuf-javanano:3.0.0-alpha-7'
         } else {
-            compile "org.gearvrf:backend_daydream:$gearvrfVersion"
-        }
-        compile 'com.google.vr:sdk-base:1.10.0'
-        compile 'com.google.protobuf.nano:protobuf-javanano:3.0.0-alpha-7'
-    } else {
-        if (rootProject.hasProperty("no_oculus")) {
-            compile(name: 'backend-debug', ext: 'aar')
-        } else { //default oculus+daydream
-            if(rootProject.useLocalDependencies) {
+            if (rootProject.hasProperty("no_oculus")) {
+                compile(name: 'backend-debug', ext: 'aar')
+            } else { //default oculus+daydream
                 compile(name: 'backend_daydream-debug', ext: 'aar')
                 compile(name: 'backend_oculus-debug', ext: 'aar')
-            } else {
-                compile "org.gearvrf:backend_daydream:$gearvrfVersion"
-                compile "org.gearvrf:backend_oculus:$gearvrfVersion"
+                compile 'com.google.vr:sdk-base:1.10.0'
+                compile 'com.google.protobuf.nano:protobuf-javanano:3.0.0-alpha-7'
             }
+        }
+    }
+} else if(rootProject.hasProperty("buildGvrfAndDemos")) {
```

Comment:
Do we need another gradle property here?

Now that we have the builds up on jcenter/mavencentral, couldn't we just default to them if it cannot find the framework at the default relative path. Something along these lines:

https://github.com/rahul27/GearVRf-Demos/commit/3b4b2277e1bb4e913529eb80a38fcb2b0ee48362

It would make the builds easier without the need to provide the extra build properties needed for gradle. It also means we could get rid of the gearvrf-libs folder.